README.md

```diff
@@ -219,10 +219,24 @@ In same cases you might need to show only the FT logo in the header, and hide al
 }
 ```
 
-#### Header with not-clickable logo, and hide footer
+##### Header with not-clickable logo, and hide footer
 If your page will be linked to from the iOS app, and no outbound navigation from it is allowed, then the flag 'hideOutboundLinks' will be set to true for you.
 This will render the header logo without it being a link, and hide the page footer.
 
+#### Footer modifications
+Similar nUi controls exist for footer. Current configuration allows footer theme changes and option disabling.
+```javascript
+{
+    nUi: {
+        footer: {
+            themeLight: 'true',
+            menu:       'no-menu',
```

Comment:
Instead of `no-menu` and `legal=true` could there be a single `filterMenu: "Legal & Privacy"` parameter? This could make it more reusable